--- vendor/upstream/libgitg/gitg-diff-view-file-renderer-textable.vala	2023-08-03 13:36:24.000000000 +0200
+++ src/vendor-gitg/libgitg/gitg-diff-view-file-renderer-textable.vala	2026-08-10 19:51:27.881472524 +0200
@@ -17,7 +17,7 @@
  * along with gitg. If not, see <http://www.gnu.org/licenses/>.
  */
 
-interface Gitg.DiffViewFileRendererTextable : DiffSelectable, DiffViewFileRenderer
+interface Gitg.DiffViewFileRendererTextable : DiffViewFileRenderer
 {
 	public abstract bool wrap_lines { get; set; }
 	public abstract new int tab_width { get; set; }
